_shows/17_18/collaborators.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Collaborators
+playwright: John Hodge
+period: Autumn
+season: In House
+season_sort: 110
+date_start: 2017-11-29
+date_end: 2017-12-02
+venue: New Theatre
+
+cast:
+- role: Mikhail Bulgakov
+  name: Callum Walker
+- role: Yelena Bulgakov
+  name: Margaux Valarche
+- role: Vladimir
+  name: Alex Piechowski
+- role: Stepan
+  name: Chris Odulele
+- role: Vasilly
+  name: Jack Lahiff
+- role: Praskovya
+  name: Rose Edgeworth
+- role: Sergei
+  name: Nat Henderson
+- role: Grigory
+  name: Bertie Beeching
+- role: Anna
+  name: Leonora Hamilton-Shield
+- role: Eva
+  name: Amy Crighton
+- role: Anton/Lagrange/Driver
+  name: Reilly Salmon
+- role: Decalev/Doctor
+  name: Matteo Bagaini
+- role: Joseph Stalin
+  name: Jack Ellis
+  note: a dictator aged 59 and 3/4
+
+crew:
+- role: Director
+  name: Will Berrington
+- role: Producer
+  name: Josie Hayden
+- role: Creative Advisor
+  name: Dave Porter
+- role: Technical Director
+  name: Darcey Graham
+- role: Lighting Designer
+  name: Nathan Penney
+- role: Sound Designer
+  name: Hannah Burne
+- role: Set Designer
+  name: Laura Wolczyk
+- role: Publicity Manager
+  name: Jonathan Taylor Davies
+- role: Shadow Technical Director
+  name: Hannah McCullough
+- role: Technical Operator
+  name: Rohanna Brown
+- role: Technical Operator
+  name: Ben Woodford
+- role: Technical Operator
+  name: Nathan Penney
+- role: Technical Operator
+  name: Maddy Strauss
+- role: Technical Operator
+  name: James Curling
+- role: Technical Operator
+  name: Georgi Paxton
```

Comment:
Had to cancel, your call as to whether to remain.

Reply:
Would remove personally